src/constants.jl

```diff
@@ -59,51 +40,6 @@ const YQ_y1(::Type{Float64}) = (
     2.35564092943068577943E5, 5.94301592346128195359E2,
     1.00000000000000000000E0
 )
-const RP_j1(::Type{Float64}) = (
-    3.68295732863852883286E15, -7.27494245221818276015E13,
-    4.52228297998194034323E11, -8.99971225705559398224E8
-)
-const RQ_j1(::Type{Float64}) = (
-    5.32278620332680085395E18, 8.95222336184627338078E16,
-    7.84369607876235854894E14, 4.74914122079991414898E12,
-    2.21511595479792499675E10, 8.35146791431949253037E7,
-    2.56987256757748830383E5, 6.20836478118054335476E2,
-    1.00000000000000000000E0
-)
-const PP_j1(::Type{Float64}) = (
```

Comment:
Unfortunately several of these constants are used for the `bessely` implementations.

Reply:
it will work out once we replace those implementations as well :)

Reply:
O most definitely!! Didn't know if you wanted to do that in separate PR or all at once 😃